radio/src/translations/de.h

```diff
@@ -927,24 +927,24 @@
 #define TR_TRAINER             		   "DSC Buchse PPM In/Out"
 #define TR_CHANS                       "Chans"
 #define TR_ANTENNAPROBLEM     		   CENTER "TX-Antennenproblem!"
-#define TR_MODELIDUSED                 "ID used in:"
-#define TR_MODELIDUNIQUE               "ID is unique"
+#define TR_MODELIDUSED                 "ID benutzt in:"
+#define TR_MODELIDUNIQUE               "ID ist eindeutig"
 #define TR_MODULE             		   "Modul-Typ"
 #define TR_RX_NAME                     "Rx Name"
 #define TR_TELEMETRY_TYPE      		   TR("Typ", "Telemetrietyp")
 #define TR_TELEMETRY_SENSORS  		   "Sensoren"
 #define TR_VALUE               		   "Wert"
-#define TR_REPEAT                      "Repeat"
-#define TR_ENABLE                      "Enable"
-#define TR_TOPLCDTIMER        		   "Top LCD Timer"
+#define TR_REPEAT                      "Wiederholung"
+#define TR_ENABLE                      "Aktivieren"
+#define TR_TOPLCDTIMER        		   "oberer LCD Timer"
 #define TR_UNIT                        "Einheit"
 #define TR_TELEMETRY_NEWSENSOR         INDENT "Sensor hinzufügen ..."
 #define TR_CHANNELRANGE                TR(INDENT "Kanäle", INDENT "Ausgangs Kanäle")  //wg 9XR-Pro
-#define TR_AFHDS3_RX_FREQ              TR("RX freq.", "RX frequency")
-#define TR_AFHDS3_ONE_TO_ONE_TELEMETRY TR("Unicast/Tel.", "Unicast/Telemetry")
+#define TR_AFHDS3_RX_FREQ              TR("RX Freq.", "RX Frequenz")
+#define TR_AFHDS3_ONE_TO_ONE_TELEMETRY TR("Unicast/Tel.", "Unicast/Telemetrie")
 #define TR_AFHDS3_ONE_TO_MANY          "Multicast"
-#define TR_AFHDS3_ACTUAL_POWER         TR("Act. pow", "Actual power")
-#define TR_AFHDS3_POWER_SOURCE         TR("Power src.", "Power source")
+#define TR_AFHDS3_ACTUAL_POWER         TR("Akt.Leis", "Aktuelle Leistung")
```

Comment:
Isn't "actual" rather "echte"/"wirkliche"?

Reply:
yes that's correct, but "echte" or "wirkliche" is also "aktuelle". I think this is best translation for that.

Reply:
@froqstar what should we do? Leave it or change it to "echte Leistung" ?

Reply:
I think "tatsächlich" would be a better translation in this context.

Reply:
@froqstar  okay. Will changed to
TR("Tat.Leis", "tatsäch. Leistung")
Is that okay?